app/src/lib/parser/index.js

```diff
@@ -43,7 +43,7 @@ export function parse_for_test(text) {
 		rules_applier(SYNTAX_RULES),
 		rules_applier(PART_OF_SPEECH_RULES),
 		rules_applier(TRANSFORM_RULES),
-		rules_applier(CHECKER_RULES),
+		rules_applier(CHECKER_RULES.slice(0,4)),	// skip 'no lookup' test until e2e testing is setup
```

Comment:
```suggestion
		rules_applier(CHECKER_RULES.slice(0,4)),	// TODO: skip 'no lookup' test until e2e testing is setup
```

so it will show up on TODO reports (see vscode extension Gruntfuggly.todo-tree)